recipes/virtualenv/build.sh

```diff
@@ -0,0 +1,4 @@
+
+#!/bin/bash
+
+$PYTHON setup.py install
```

Comment:
`setuptools` is only used if it's on SYSPATH.  Otherwise `setup.py` falls back to distutils.